docs/src/main/asciidoc/mongodb.adoc

```diff
@@ -598,6 +606,13 @@ Currently, Quarkus doesn't support the `mongodb+srv` protocol in native mode.
 link:https://docs.mongodb.com/manual/core/security-client-side-encryption/[Client-Side Field Level Encryption] is also not supported in native mode.
 ====
 
+[TIP]
+====
+If you encounter the following error when running your application in native mode: +
+`Failed to encode 'MyObject'. Encoding 'myVariable' errored with: Can't find a codec for class org.acme.MyVariable.` +
+This means that the `org.acme.MyVariable` class is not known to GraalVM, to fix this you can add the `@RegisterForReflection` annotation to your `MyVariable` class.
```

Comment:
the remedy is to add the `@RegisterForReflection` annotation to your `MyVariable` class.